src/core/lib/gprpp/atomic.h

```diff
@@ -21,10 +21,60 @@
 
 #include <grpc/support/port_platform.h>
 
-#ifdef GPR_HAS_CXX11_ATOMIC
-#include "src/core/lib/gprpp/atomic_with_std.h"
-#else
-#include "src/core/lib/gprpp/atomic_with_atm.h"
-#endif
+#include <atomic>
+
+namespace grpc_core {
+
+// Prefer the helper methods below over the same functions provided by
+// std::atomic, because they maintain stats over atomic opertions which are
+// useful for comparing benchmarks.
+
+template <typename T>
+bool compare_exchange_weak(std::atomic<T>* storage, T& expected, T desired,
```

Comment:
Should these have C++-style names?

Reply:
Sure, I added `Atomic` prefix to all these methods. Also changed the type of `expected` to `T*` from `T&` so that it conforms to the style.